sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/SparkSQLParser.scala

```diff
@@ -66,7 +66,13 @@ class SqlLexical(val keywords: Seq[String]) extends StdLexical {
 
   override lazy val token: Parser[Token] =
     ( identChar ~ (identChar | digit).* ^^
-      { case first ~ rest => processIdent((first :: rest).mkString) }
+      { case first ~ rest => 
+          val rsIdent = processIdent((first :: rest).mkString.toLowerCase)
```

Comment:
@chenghao-intel 
Since Keyword is a `constant` and it's usage is to parsing matching and identify others, we don't need them after parsing correctly, so here I only make `Keyword` lower case is doesn't matter and will not cause other issues.